debian/control

```diff
@@ -54,11 +54,14 @@ Architecture: any
 Depends: adduser,
          apparmor (>= 2.10.95-0ubuntu2.2),
          ca-certificates,
+         cgroup-lite,
          gnupg1 | gnupg,
          snap-confine (= ${binary:Version}),
          squashfs-tools,
-         systemd,
+         sudo (>=1.8.9p5-1ubuntu1.3),
```

Comment:
Agreed, we should investigate into a cleaner solution. I would rather like to postpone that and left a comment instead. I hope that's okay with you.